modules/database/rds/stack.py

```diff
@@ -0,0 +1,139 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import json
+from typing import Any, cast
+
+import aws_cdk as cdk
+import cdk_nag
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_rds as rds
+from aws_cdk import aws_secretsmanager as sm
+from constructs import Construct, IConstruct
+
+
+def _get_hosted_rotation_for_engine(engine: str) -> sm.HostedRotation:
+    if engine == "mysql":
+        return sm.HostedRotation.mysql_single_user()
+    elif engine == "postgresql":
+        return sm.HostedRotation.postgre_sql_single_user()
+    else:
+        raise ValueError(f"Unsupported engine: {engine}")
+
+
+def _get_db_instance_engine(engine: str) -> rds.IInstanceEngine:
+    if "mysql" == engine:
+        return rds.DatabaseInstanceEngine.mysql(version=rds.MysqlEngineVersion.VER_8_0_35)
```

Comment:
Do we want to allow specify the version via `rds.MysqlEngineVersion.of(str)`? 

The use case I am thinking of is assuming we keep db versions up-to-date, the user wants to consume a new version of the module, without having to migrate to the latest version of the database.

Reply:
I think that's a good idea! I'll make the change.